app/uk/gov/hmrc/fileupload/quarantine/QuarantineRepository.scala

```diff
@@ -60,7 +60,8 @@ object EnvelopeReport{
 
 case class EnvelopeConstraints(maxItems: Int,
                                maxSize: String,
-                               maxSizePerItem: String)
+                               maxSizePerItem: String,
+                               allowZeroLengthFiles: Option[Boolean])
```

Comment:
safer to have default as it is new/migrated field?

Reply:
discussed offline